packages/patrol/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
-## Unreleased
-
-- Add `--build-name` and `--build-number` flags to `patrol test`, `patrol develop`, and `patrol build` commands.
+## 4.0.0
+
+- **Breaking**: Change default test directory from `integration_test` to `patrol_test`. (#2728)
+- Add support for configurable test directory via `test_directory` option in `pubspec.yaml`. (#2728)
+- Add `--build-name` and `--build-number` flags to `patrol test`, `patrol develop`, and `patrol build` commands. (#2590)
+- Replace clearing permissions functionality with uninstalling app on iOS. (#2730)
+- Bump `patrol_log` to `0.6.0`.
+- Change default frame policy to `LiveTestWidgetsFlutterBindingFramePolicy.fullyLive`. (#2763)
```

Comment:
- Missing entry about web support, introducing PlatformAutomator and NativeAutomator(2) deprecation